### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by nlukic97**
Added the requested changes for the Profile model, and it's methods.

### CODE REVIEW

**Comment by nlukic97** [comment]
Created a model association between **User** and **Profile** in the form of a **OneToOne** relationship - will update if necessary.

https://github.com/nlukic97/uteam-api/blob/db0bebcecca85da6f580460aefe977db10d10e1b/ts/models/User.ts#L50-L52

File: ts/controllers/Profile.controller.ts [hunk 1]
```diff
@@ -0,0 +1,206 @@
...
+        const profileExists = await Profile.findOne({where:{
+            name: submitData.name
+        }})
+        if(profileExists) throw 'A profile with this name already exists. Please enter another name.'
+
+        
+        // constructing the profile
+        const profile = await Profile.build(submitData)
+        
+        profile.save().then(()=>{
```
Comment (vuxor) [inline ts/controllers/Profile.controller.ts:74]: You used async/await previously in this function. Try to convert promise.

Comment (nlukic97) [reply ts/controllers/Profile.controller.ts]: https://github.com/nlukic97/uteam-api/blob/a2ce8efe4c1a75b7089404d55f58539788439279/ts/controllers/Profile.controller.ts#L72

Fixed - removed await and used the "create" method which returns a promise.

**Comment by nlukic97** [comment]
Added README instructions on setting up ACCESS_TOKEN_SECRET in .env needed for JWT token creation. 
Also added a check for this .env variable in **app.ts**:
https://github.com/nlukic97/uteam-api/blob/815edc0a23b61e4e5a92a40721e771cd21feba2d/ts/app.ts#L8-L9

Without this check, the server can run without the previously mentioned .env variable, but will eventually crash when trying to create a JWT token for a newly registered user.

**Pull request merged**